src/main/java/com/softserve/paymentservice/service/PaymentService.java

```diff
@@ -0,0 +1,93 @@
+package com.softserve.paymentservice.service;
+
+import com.softserve.paymentservice.model.Card;
+import com.stripe.Stripe;
+import com.stripe.exception.StripeException;
+import com.stripe.model.Charge;
+import com.stripe.model.Customer;
+import com.stripe.model.Refund;
+import com.stripe.model.Token;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+@Service
+public class PaymentService {
+
+    @Value("${stripe.key.secret}")
+    private String stripeSecretKey;
+
+
+    public Boolean pay(int amount, Card card, String cvc) throws StripeException {
+        return getCharge(amount, card, cvc).getPaid();
+    }
+
+    public Boolean checkingPayment(Card card, String cvc) throws StripeException {
```

Comment:
don't rethrow StripeException, controllers shouldn't know about payments provider, which you use